os/include/debug.h

```diff
@@ -725,6 +725,22 @@ Once LOGM is approved, each module should have its own index
 #define tmvdbg(...)
 #endif
 
+#ifdef CONFIG_DEBUG_PREFERENCE
```

Comment:
You didn't add definitions on #else case of CONFIG_CPP_HVAE_VARARGS.

Reply:
Done